### PR TITLE
chore(litmus-portal): Added subject and context fields while workflow creation and fixed some minor bugs

### PR DESCRIPTION
**Pull request by amityt**
Signed-off-by: Amit Kumar Das <amit@chaosnative.com>

## Proposed changes

This PR adds:
- Subject field in the workflow maninfest
- Context field for chaos engines
- Fixed name and description change bug in verify and commit page
- Minor bug fixes

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/index.tsx [hunk 1]
```diff
@@ -222,8 +231,18 @@ const TuneWorkflow = forwardRef((_, ref) => {
       .then((data) => {
         data.text().then((yamlText) => {
           const wfmanifest = updateEngineName(YAML.parse(yamlText));
+          const updatedManifest = YAML.parse(wfmanifest);
+          updatedManifest.spec.arguments.parameters.forEach(
+            (parameter: any, index: number) => {
+              if (parameter.name === 'adminModeNamespace') {
```
Comment (Saranya-jena) [inline litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/index.tsx:237]: Can you check if it's possible/required to convert 'adminModeNamespace' into a constant? 🤔

File: litmus-portal/frontend/src/views/CreateWorkflow/VerifyCommit/index.tsx [hunk 2]
```diff
@@ -350,6 +374,37 @@ const VerifyCommit = forwardRef(
                   />
                 </div>
               </div>
+
+              <div className={classes.summaryDiv}>
+                <div className={classes.innerSumDiv}>
+                  <div style={{ display: 'flex' }}>
```
Comment (Saranya-jena) [inline litmus-portal/frontend/src/views/CreateWorkflow/VerifyCommit/index.tsx:380]: Can you remove this inline styling and mode it to styles folder?

File: litmus-portal/frontend/src/views/CreateWorkflow/WorkflowSettings/index.tsx [hunk 3]
```diff
@@ -215,6 +216,14 @@ const WorkflowSettings = forwardRef((_, ref) => {
...
+              <InputField
+                InputProps={{
+                  readOnly: true,
+                }}
+                className={classes.nsInput}
+                label={t('createWorkflow.chooseWorkflow.label.namespace')}
+                value={workflowData.namespace}
+              />
             </div>
             <div aria-details="spacer" style={{ margin: '3rem 0' }} />
```
Comment (Saranya-jena) [inline litmus-portal/frontend/src/views/CreateWorkflow/WorkflowSettings/index.tsx:228]: same here 🙂

**Comment by Saranya-jena** [comment]
There's a minor deepscan issue found, can you fix it?

**Review by Saranya-jena — APPROVED** [review]

File: litmus-portal/frontend/src/views/CreateWorkflow/WorkflowSettings/styles.ts [hunk 4]
```diff
@@ -112,6 +112,16 @@ const useStyles = makeStyles((theme) => ({
...
   },
+  nsInput: {
+    marginLeft: theme.spacing(2.5),
+  },
+  mainDiv: {
+    display: 'flex',
+    justifyContent: 'space-between',
+  },
+  descDiv: {
+    margin: theme.spacing(6, 0, 0, 0),
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/CreateWorkflow/WorkflowSettings/styles.ts:123]: Use marginTop

**Review by ishangupta-ds — CHANGES_REQUESTED** [review]

File: litmus-portal/frontend/src/views/CreateWorkflow/VerifyCommit/index.tsx (hunk 2 above)
Comment (ishangupta-ds) [reply litmus-portal/frontend/src/views/CreateWorkflow/VerifyCommit/index.tsx]: @Saranya-jena for single css property this way is better. cc: @amityt

File: litmus-portal/frontend/src/views/CreateWorkflow/WorkflowSettings/index.tsx (hunk 3 above)
Comment (ishangupta-ds) [reply litmus-portal/frontend/src/views/CreateWorkflow/WorkflowSettings/index.tsx]: Here its not okay as spacing should be used from theme for all margin and paddings.

**Review by ishangupta-ds — APPROVED** [review]
LGTM

**Review by S-ayanide — APPROVED** [review]

**Pull request merged**